rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -270,13 +271,29 @@ public RMState getState(@HeaderParam("sessionid") String sessionId) throws NotCo
     @GZIP
     @Path("monitoring")
     @Produces("application/json")
-    public RMStateDelta getInitialState(@HeaderParam("sessionid") String sessionId,
+    public RMStateDelta getRMStateDelta(@HeaderParam("sessionid") String sessionId,
             @HeaderParam("clientCounter") @DefaultValue("-1") String clientCounter) throws NotConnectedException {
         checkAccess(sessionId);
         long counter = Integer.valueOf(clientCounter);
         return RMStateCaching.getRMInitialState(counter);
```

Comment:
You could also rename this method accordingly (e.g. `RMStateCaching.getRMStateDelta(counter)`), as you wish.

Reply:
Thanks. Applied